production.ini

```diff
@@ -442,6 +442,15 @@ fedora_epel.mandatory_days_in_testing = 14
 #f28.post_beta.critpath.num_admin_approvals = 0
 #f28.post_beta.critpath.min_karma = 2
 
+###
+### Automatic updates for releases
+###
+
+# You can specify that updates are automatically created for a release when
```

Comment:
I think we don't need this setting any more

Reply:
I'll remove this...